dv-export/python/query-changes/README.md

```diff
@@ -86,16 +83,13 @@ The same data is presented in a table format with dates:
 | 2024-03-20 |           |           | V2        |             |
 | 2024-04-01 |           |           |           | DV3         |
 
-### Restate Mode Example:
+### Restate Mode Example
 
-In `restate` mode, the application identifies analytic object instances that have changed between
-the specified data versions provided by the user.
-For each of these changed instances, it retrieves the entire history.
-Because the full history will be retrieved, the results will vary depending on when the application was launched.
+Use `restate` mode to fetch the full history of a record that changed between two data versions. You may get a different result depending when you run the sample application.
```

Comment:
It can be challenging to explain everything in one sentence. That’s why I’ve broken it down into several steps. 

First, we identify instances of the analytic object that have been modified. Then, we retrieve the full history for those changed instances.